src/components/Explorer.js

```diff
@@ -64,20 +64,21 @@ class Explorer extends React.Component {
 
   componentDidMount() {
     const { appState, dispatch, router } = this.props
+    const { since, until } = appState.filters
+    const { query } = router.location
+
+    const clean = (val, alt) => {
+      const yr = +val
+      return yr >= 1960 && yr <= 2014 ? yr : alt
+    }
+
     const filters = {
       ...this.props.filters,
       ...router.params,
-      ...router.location.query,
+      since: clean(query.since, since),
```

Comment:
this only gets called on the initial load, so i don't think we have to worry about that; also, i believe this is how it worked before